pkg/apis/etcd/v1beta2/cluster.go

```diff
@@ -99,6 +101,15 @@ type ClusterSpec struct {
 
 	// etcd cluster TLS configuration
 	TLS *TLSPolicy `json:"TLS,omitempty"`
+
+	// busybox:latest uses uclibc which contains a bug that sometimes prevents name resolution
```

Comment:
We shouldn't bloat the top level spec.

I would suggest adding these to PodPolicy.

Reply:
@hongchaodeng OK. I'll do that. 
Any other suggestions while i'm on it?